compiler/plugins/target/AMD-AIE/iree-amd-aie/IR/AMDAIEAttrs.td

```diff
@@ -162,4 +162,15 @@ def AMDAIE_PackingConfigAttr :
   }];
 }
 
+def AMDAIE_NumInputLoopsAttr :
```

Comment:
Documentation! Or maybe it can be removed... can the number of loops be inferred JIT?